app/controllers/analytics_controller.rb

```diff
@@ -1,6 +1,4 @@
 class AnalyticsController < ApplicationController
-  before_filter :authenticate, :employee?
```

Comment:
If nothing else uses these methods we should strip them out in a clean-up MR later to simplify the application controller

Reply:
👍 Yeah, the existing `authenticate` method should probably fully go away